packages/protocol/test/common/integration.ts

```diff
@@ -414,7 +414,7 @@ contract('Integration: Exchange', (accounts: string[]) => {
       originalGold = await goldToken.balanceOf(accounts[0])
       originalReserve = await goldToken.balanceOf(reserve.address)
       await goldToken.approve(exchange.address, sellAmount)
-      await exchange.exchange(sellAmount, minBuyAmount, true)
+      await exchange.sell(sellAmount, minBuyAmount, true)
```

Comment:
Could `exchange.buy` also be tested here?

Reply:
good idea, will do that